src/Core/Manifest/Module.php

```diff
@@ -19,6 +20,23 @@ class Module implements Serializable
      */
     const TRIM_CHARS = ' /\\';
 
+    /**
+     * Return value of getCILibrary() when module uses PHPUNit 9
+     */
+    const CI_PHPUNIT_NINE = 'PHPUnit9';
```

Comment:
Use the standard `const CI_PHPUNIT_NINE = 'CI_PHPUNIT_NINE';`

Reply:
Done